tests/scripts/all.sh

```diff
@@ -3505,6 +3505,11 @@ component_build_armcc () {
     # ARM Compiler 6 - Target ARMv8.2-A - AArch64
     armc6_build_test "-O1 --target=aarch64-arm-none-eabi -march=armv8.2-a+crypto"
 }
+support_build_armcc () {
+    arm5_cc="$ARMC5_BIN_DIR/armcc"
+    arm6_cc="$ARMC6_BIN_DIR/armclang"
+    `check_tools "$arm5_cc" "$arm6_cc" > /dev/null 2>&1`
```

Comment:
This tries to execute the output of `check_tools`. Currently I think the output is always empty, and an empty command does nothing, so it works, but it's fragile.

```suggestion
    (check_tools "$ARM5_CC" "$ARM6_CC" > /dev/null 2>&1)
```

Reply:
Thanks for your suggestion! I was thinking of this but had no idea at the moment I was coding this. I will do the change.